### PR TITLE
fuse sidecar yaml params

### PR DESCRIPTION
**Pull request by ransomw1c**

### CODE REVIEW

**Comment by ransomw1c** [comment]
resurrecting #270 for fuse-only with context debug and ci tests in-place

**Comment by ransomw1c** [comment]
going to add building yaml from golang as part of the demo and rm the wip tag..

File: hack/k8s/example-coord.template.yaml [hunk 1]
```diff
@@ -69,8 +72,14 @@ spec:
         - mountPath: /tmp/gac
           name: google-application-credentials
         env:
+
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /tmp/gac/google-application-credentials.json
+
+        - name: dm_fuse_params
+          value: /fuse-params.yaml
```
Comment (galvare2) [inline hack/k8s/example-coord.template.yaml]: Where does this file come from in this example? Is it built into the datamon sidecar container?

File: hack/fuse-demo/wrap_datamon.sh [hunk 2]
```diff
@@ -20,113 +31,236 @@ POLL_INTERVAL=1 # sec
...
+    env_vars_file=/tmp/env_vars_from_params.sh
+    if [[ -f $dm_fuse_params ]]; then
+        dbg_print "parsing parameters YAML file $dm_fuse_params"
+        dm_fuse_params_val=$(cat $dm_fuse_params)
+    else
+        dbg_print "parsing parameters YAML from env var"
+        dm_fuse_params_val=$dm_fuse_params
+    fi
+    print -- $dm_fuse_params_val | \
+        datamon_sidecar_param parse fuse > $env_vars_file
```
Comment (galvare2) [inline hack/fuse-demo/wrap_datamon.sh]: If I understand correctly, you've implemented a new datamon command-line endpoint, invoked here, that deserializes the yaml into a set of env vars that are saved in the env vars file. Is that correct?

File: hack/k8s/example-coord.template.yaml (hunk 1 above)
Comment (ransomw1c) [reply hack/k8s/example-coord.template.yaml]: it's built via `hack/fuse-demo/write_fuse_params.go` ...

... external to docker then there's an `ADD` in the dockerfile.

let's sync up re. pipelines tool to find out how to tighten up integration.

File: hack/fuse-demo/wrap_datamon.sh (hunk 2 above)
Comment (ransomw1c) [reply hack/fuse-demo/wrap_datamon.sh]: yes, although this is an implementation detail that's intended to be opaque from your review

**Pull request merged**